metrics/sinks/influxdb/influxdb.go

```diff
@@ -107,7 +110,11 @@ func (sink *influxdbSink) ExportData(dataBatch *core.DataBatch) {
 
 			dataPoints = append(dataPoints, point)
 			if len(dataPoints) >= maxSendBatchSize {
-				sink.sendData(dataPoints)
+				sink.Add(1)
+				sink.conChan <- struct{}{}
```

Comment:
this needs more comments -- it's not immediately clear that it's being used to limit the number of sends that can go on at once

Reply:
Have add some comments in `wg` and `conChan` fields. :)